index.js

```diff
@@ -127,6 +127,11 @@ MochaWrapper.prototype.describe = function describe(msg, fn) {
 	createAssertion('describe', msg, wrappers, fn);
 };
 
+MochaWrapper.prototype.context = function context(msg, fn) {
+	var wrappers = getThisWrappers(checkThis(this));
+	createAssertion('context', msg, wrappers, fn);
```

Comment:
since it's an alias, should this just do `MochaWrapper.prototype.context = MochaWrapper.prototype.describe;`?